string.md

```diff
@@ -0,0 +1,54 @@
+---
+title: STRING
+toc: true
+---
+
+## Description
+
+The `STRING` [data type](data-types.html) stores strings of variable, unlimited length. 
+
+## Synonyms 
+
+In CockroachDB, the following are synonyms of `STRING` and are implemented identically: 
+
+- `CHAR` 
+- `CHAR(n)` 
+- `VARCHAR`
+- `VARCHAR(n)` 
+- `TEXT`
+
+## Length
+
+Length is always variable and unlimited for a `STRING` value. If you specify a fixed length by declaring `CHAR(n)` or `VARCHAR(n)` on a column, it will not be enforced.
```

Comment:
If it's ok, I'll just remove "unlimited" and keep as-is for now. Once we've fixed this, I'll updated.